mava/adders/reverb/sequence.py

```diff
@@ -20,43 +20,45 @@
 This implements adders which add sequences or partial trajectories.
 """
 
-from typing import NamedTuple, Optional
+import operator
+from typing import Optional
 
 import reverb
 import tensorflow as tf
 import tree
-from acme import specs, types
+from acme import specs
 from acme.adders.reverb import utils
-from acme.utils import tree_utils
+from acme.adders.reverb.sequence import SequenceAdder
+from acme.types import NestedSpec
 
 from mava.adders.reverb import base
-from mava.adders.reverb import utils as mava_utils
+from mava.adders.reverb.base import ReverbParallelAdder
+from mava.adders.reverb.utils import trajectory_signature
 
+# TODO Clean this up, when using newer versions of acme.
```

Comment:
Are you referring to the version `0.2.1` here or newer than that, i.e. future versions?

Reply:
Future versions. 

In the latest codebase (not yet a package), `EndOfEpisodeBehavior` was renamed to `EndBehavior`. If we simply import `EndOfEpisodeBehavior` , it will break in `0.2.2`. Importing `EndBehavior` currently breaks since it was added after `0.2.1`.